code/datums/wires/smartfridge.dm

```diff
@@ -43,6 +39,7 @@
 		if(WIRE_THROW)
 			S.shoot_inventory = !S.shoot_inventory
 		if(WIRE_SHOCK)
+			S.shock(usr, 50)
```

Comment:
`usr` isn't acceptable, this proc receives the `user` already as a parameter, adjust the proc signature and use that instead